scripts/qa_crowbarsetup.sh

```diff
@@ -3173,6 +3173,18 @@ function set_noproxyvar
     no_proxy="${no_proxy%,}";
 }
 
+# commit a proposal, but use crowbarctl from cloud6 on
+function crowbar_proposal_commit
+{
+    local proposal="$1"
+    local proposaltype="${2:-default}"
+    if iscloudver 6plus ; then
```

Comment:
We only want to run this if crowbarctl is new enough.. so GM6, GM6+up and so on are currently not good?